include/zephyr/dt-bindings/gpio/realtek-gpio.h

```diff
@@ -39,4 +39,112 @@
 /** Interrupt Mask since rts5912 does not support GPIO_INT_LEVELS_LOGICAL*/
 #define RTS5912_GPIO_INTR_MASK   (1U << 21 | 1U << 22 | 1U << 24 | 1U << 25 | 1U << 26)
 
+/** GPIO number map between spec and zephyr */
+#define RTS5912_GPIO_NUM_000 &gpioa 0
```

Comment:
Considering the symbol name is just `GPIO000`, I suggest dropping the `_NUM_` designator.
```suggestion
#define RTS5912_GPIO000 &gpioa 0
```